src/config.ts

```diff
@@ -59,15 +56,21 @@ interface BrokerOptions {
   // Broker type.
   // Default value is "orion".
   type?: "orion" | "kafka";
-
-  kafka?: KafkaOptions;
 }
 
 // Device manager options
 interface DeviceManagerOptions {
   // Device manager address.
   // This attribute includes port, such as localhost:4000.
   host: string;
```

Comment:
Is the device manager host address still required if the data is received via kafka?

Reply:
No, it isn't.